rskj-core/src/main/java/co/rsk/remasc/Remasc.java

```diff
@@ -115,11 +115,11 @@ void processMinersFees() {
         int uncleGenerationLimit = constants.getUncleGenerationLimit();
         Deque<Map<Long, List<Sibling>>> descendantsBlocks = new LinkedList<>();
 
-        // this search can be optimized if have certainty that the execution block is not in a fork
-        // larger than depth
-        Block currentBlock = blockStore.getBlockByHashAndDepth(
-                executionBlock.getParentHash().getBytes(),
-                remascConstants.getMaturity() - 1 - uncleGenerationLimit
+        // this search is noe optimized if have certainty that the execution block is not in a fork
```

Comment:
Is this change on the comment intended?

Reply:
Yes, the algorithm supports the existence of a fork, but it could be degraded to the original algorithm performance if the fork involves > blocks than the remasc maturity parameter